tls/s2n_connection.c

```diff
@@ -1224,6 +1224,9 @@ S2N_CLEANUP_RESULT s2n_connection_apply_error_blinding(struct s2n_connection **c
         return S2N_RESULT_OK;
     }
 
+    /* Ensure that conn->in doesn't contain any leftover invalid or unauthenticated data. */
+    RESULT_GUARD_POSIX(s2n_stuffer_wipe(&(*conn)->in));
```

Comment:
Do we need to wipe anything else since we're making this wipe apply more generically? Like s2n_record_wipe() instead? like doesn't this function get called potentially after writes as well 🤔 ?